### ISSUE TITLE
Run integration tests in parallel

### ISSUE THREAD

**Issue by slifty** [PhilanthropyDataCommons/service#610]
We have had to use `-runInBand` due to a limitation with our migration library's lack of support for multiple schemas, which means our integration tests don't run in parallel.  How sad!

Luckily once #609 is addressed we can remove this limitation.  This *also* means we can remove the patch we wrote for #462, since that CI bug only happens when `--runInBand` is set.

**Comment by slifty** [comment]
I spent a small time box on this -- removing --runInBand seems to cause a different type of performance issue, resulting in test timeouts.  Confusingly, it also results in a warning of `A worker process has failed to exit gracefully and has been force exited. This is likely caused by tests leaking due to improper teardown. Try running with --detectOpenHandles to find leaks. Active timers can also cause this, ensure that .unref() was called on them.`

When I run the tests with `--detectOpenHandles` it is identifying "timeouts" in very strange places -- e.g. for database teardown, or for queries within tests, both of which I believe are being correctly awaited within `async` functions.  Which is to say, there is something that changes in how Jest runs tests when they aren't being run in band.

I don't think is worth trying to understand in this moment.  I'll leave the issue open in case we want to prioritize it later.

**Comment by reefdog** [comment]
I just had a test fail due to a timeout issue. I'm putting the info here, although I'm not sure if it's actually related. (Running tests again, it succeeded.)

```
 FAIL  src/database/operations/load/__test__/loadBundle.int.test.ts (5.419 s)
  ● loadBundle › Should return a bundle for the query

    thrown: "Exceeded timeout of 5000 ms for a hook.
    Use jest.setTimeout(newTimeout) to increase the timeout value, if this is a long-running test."

      27 | });
      28 |
    > 29 | beforeEach(async () => {
         | ^
      30 | 	mockJwks.start();
      31 | 	await prepareDatabaseForCurrentWorker();
      32 | 	await createTestUser();

      at Object.<anonymous> (src/test/integrationSuiteSetup.ts:29:1)
```

**Comment by slifty** [comment]
@reefdog that is a new issue -- it may be related, it may not be -- but you should just open an issue for it (and link to this if you think it's related!)

Would you mind opening an issue for it?

**Comment by reefdog** [comment]
Done. In my defense, you said the word "timeout" somewhere in this issue and I can't be bothered to figure out why.

**Comment by reefdog** [comment]
Texas drivers save thousands on insurance with this one easy trick
